dbus/org.nemo.ssu.xml

```diff
@@ -35,6 +35,9 @@
     <method name="isRegistered">
       <arg direction="out" type="b" name="status"/>
     </method>
+    <method name="currentDomain">
+        <arg direction="out" type="s" name="domain"/>
+    </method>
```

Comment:
I wonder if this should be just domain instead of currentDomain, we have also e.g. flavour there and not currentFlavour.